store/nosqlstore/webhook_store.go

```diff
@@ -0,0 +1,347 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
GitHub is wanting to collapse this file by default. But this is probably the most important one. (Maybe my comment here will keep it from getting collapsed.)